docs/source/poutine.rst

```diff
@@ -8,13 +8,28 @@ programs. Pyro's inference algorithms are all built by applying these handlers t
 Handlers
 ---------
 
-.. automodule:: pyro.poutine
+.. automodule:: pyro.poutine.handlers
     :members:
     :undoc-members:
     :show-inheritance:
 
+Trace
+------
+
+.. automodule:: pyro.poutine.trace
+    :members:
+    :undoc-members:
+    :show-inheritance:
 
 Messengers
 -----------
 
 .. include:: pyro.poutine.txt
```

Comment:
not part of this PR, but this file should technically be changed to `pyro.poutine.messenger`